README.md

```diff
@@ -68,7 +68,24 @@ Clear containers:
 docker rm -f $(docker ps -a -q)
 ```
 
-## Deploying to Google Cloud Platform
+### Deploying to Google Cloud Platform
 ```
 gcloud app deploy app.yaml --project <your-project-id>
-```
\ No newline at end of file
+```
+
+## Rendering
+Once you have the service up and running, you'll need to implement the differential serving
+layer. This checks the user agent to determine whether prerendering is required.
+
+### Web components
+If you are using web components v0 (deprecated), you will need to enable Shady DOM to
+render correctly. In Polymer 1.x, which uses web components v0, Shady DOM is enabled by default.
+If you are using Shadow DOM, override this by setting `?dom=shady` when directing requests
+to the bot-render service.
+
+If you are using web components v1, simply pass `wc=1` when directing requests to the bot-render
```

Comment:
Awesome, thanks for pointing that out. Changed to `wc-inject-shadydom=true`. @kevinpschaaf PTAL.